iris-logo/README.md

```diff
@@ -0,0 +1,41 @@
+# The Iris Logo
+
+## Logo
+
+The official logo can be obtained, in SVG form, via
+
+    latexmk logo
+
+This will produce the SVG by first compiling to DVI then to SVG using `dvisvgm`.
+Alternatively, by running
+
+    latexmk -pdf logo
+
+one gets a PDF and an SVG obtained by running `pdf2svg`.
+
+To clean up:
+
+    latexmk -c logo # gets rid of auxiliary files (including DVI and PDF)
+    latexmk -C logo # gets rid of SVG output too
+
+
+## Favicon
+
+Similar commands deal with the favicon:
+
+    latexmk favicon # produce favicon.svg via dvisvgm
```

Comment:
What is this witchcraft, how can latexmk know to produce ico files?^^

Do we have any control about the sizes if the images in the ico file?

Reply:
Right now it seems to be 24x24, which is larger than the old one (which I think was 16x16).

Reply:
Oh wait, it produces svg, not ico -- silly me. So `convert` now suddenly picks a different size? That seems rather fragile?

Reply:
You could make it produce the ico too by editing the `latexmkrc` file...
I don't know the details of how `convert` produces ico files, I am sure you can instruct it to produce precise sizes...